partridge/writers.py

```diff
@@ -35,14 +36,15 @@ def write_feed_dangerously(
     your own risk.
     """
     nodes = DEFAULT_NODES if nodes is None else nodes
-    try:
-        tmpdir = tempfile.mkdtemp()
+    with tempfile.TemporaryDirectory() as tmpdir:
 
         def write_node(node):
             df = feed.get(node)
             if not df.empty:
                 path = os.path.join(tmpdir, node)
-                df.to_csv(path, index=False)
+                df.to_csv(
+                    path, index=False, quoting=csv.QUOTE_NONNUMERIC, encoding="utf-8"
```

Comment:
Is `QUOTE_NONNUMERIC` necessary?

Reply:
good question. docs say:

> No automatic data type conversion is performed unless the QUOTE_NONNUMERIC format option is specified (in which case unquoted fields are transformed into floats).

so I think maybe we actually very much don't want it, unless we want floats everywhere universally. FWIW this is a pretty strange side effect.

Also I think we could do away with the entire first commit, since Pandas defaults to writing unicode CSV.

Reply:
> Also I think we could do away with the entire first commit, since Pandas defaults to writing unicode CSV.

I like the new temp directory change 👍, fine to drop the rest.